builder_childchain/Dockerfile

```diff
@@ -22,6 +22,9 @@ ENV PATH=/usr/local/otp/bin:$PATH
 ENV PATH=/usr/local/rebar3/bin:$PATH
 ENV PATH=/usr/local/elixir/bin:$PATH
 
+RUN apk add curl && curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y
```

Comment:
which version of rust is this installing?
perhaps we should split this into a builder image, just like erlang and elixir.

Reply:
In addition to what Ino said, you could consider using a SHA value to verify the binary is the officially released version in case someone has compromised `rustup.rs`.

Reply:
@InoMurko @jbunce I added the rust build image with a version specification. Also, I added rustup as a script file